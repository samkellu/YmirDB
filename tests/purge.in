set a 123 22
list entries
snapshot
set a 22 11 33 22
list entries
snapshot
set b a 123 3
list entries
snapshot
set a 44 55
list entries
snapshot
purge a
checkout 1
list entries
checkout 2
list entries
checkout 3
list entries
checkout 4
list entries
bye
