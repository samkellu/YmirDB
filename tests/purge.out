> ok

> a [123 22]

> saved as snapshot 1

> ok

> a [22 11 33 22]

> saved as snapshot 2

> ok

> b [a 123 3]
a [22 11 33 22]

> saved as snapshot 3

> ok

> b [a 123 3]
a [44 55]

> saved as snapshot 4

> not permitted

> ok

> a [123 22]

> ok

> a [22 11 33 22]

> ok

> b [a 123 3]
a [22 11 33 22]

> ok

> b [a 123 3]
a [44 55]

> bye
